wikimedia/wikipedia-ios
# [HOLD] Watchlist - add entry point in Account Settings

**Phabricator:** https://phabricator.wikimedia.org/T335606 (partially)

### Notes
Adds Watchlist list view entry point in Account Settings table for logged in users. 

### Test Steps
1. Enable Watchlist feature flag or use Staging build
2. Log in to an account
3. Confirm Watchlist appears in the Account Settings table
